staging/src/k8s.io/kube-aggregator/pkg/controllers/status/metrics.go

```diff
@@ -30,7 +30,7 @@ var (
 	)
 	unavailableGauge = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "aggregator_unavailable_apiserver_gauge",
+			Name: "aggregator_unavailable_apiservice_gauge",
```

Comment:
The convention for naming here would be `aggregator_apiserver_unavailable`, the type information already says it's a gauge.

Reply:
fixed